CSC8503/InventoryBuffSystem/FlagGameObject.cpp

```diff
@@ -0,0 +1,56 @@
+#include "FlagGameObject.h"
+#include "StateTransition.h"
+#include "StateMachine.h"
+#include "State.h"
+#include "PhysicsObject.h"
+#include "Vector3.h"
+#include "map";
```

Comment:
usual convention is to put std libraries in angle brackets, like <map>, rather than "map"